doc/nrf/test_and_optimize/optimizing/power_general.rst

```diff
@@ -181,11 +181,34 @@ Bluetooth Mesh
 
 The Bluetooth Mesh protocol offers the :ref:`ug_bt_mesh_configuring_lpn` feature for optimizing the power consumption of the Bluetooth Mesh devices.
 
+Gazell
+======
+
+Gazell is an asymmetrical protocol where the Device role is power-optimized.
+The exact current consumption depends on a number of parameters that are defined statically in the API.
+See the Constants section of the :ref:`Gazell Link Layer API documentation <nrfxlib:gzll_api>`.
+The most important parameters from the current consumption point of view are:
+
+* :c:macro:`NRF_GZLL_DEFAULT_TX_POWER` - can be used to find a trade off between communication range and current consumption.
```

Comment:
The general recommendations also mention radio TX power. Is this line duplicate?

[Configure radio transmitter power](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/test_and_optimize/optimizing/power_general.html#configure-radio-transmitter-power)

Reply:
May be, it's kind of obvious anyway. I am going to rephrase the docs a bit to refer to API functions rather than the constants. Do you have any conclusions on Gazell power optimization based on Devzone?